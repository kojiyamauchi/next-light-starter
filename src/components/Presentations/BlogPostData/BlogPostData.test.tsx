/*
  Blog Post Data Component Unit Test.
*/

import { BlogPostData } from './'
import { render /* ,screen */ } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
// import userEvent from '@testing-library/user-event'

describe('Blog Post Data Component Unit Test', () => {
  it('Snap Shot Testing', () => {
    const { asFragment } = render(<BlogPostData categories={['category-1', 'category-2']} date="2021/01/01" />)
    expect(asFragment()).toMatchSnapshot()
  })
})
