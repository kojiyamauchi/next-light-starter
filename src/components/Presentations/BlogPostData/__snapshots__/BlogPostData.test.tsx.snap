// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog Post Data Component Unit Test Snap Shot Testing 1`] = `
<DocumentFragment>
  <ul
    class="wrapper"
  >
    <li
      class="category"
    >
      <a
        class="category-link"
        href="/blog-category/category-1"
      >
        category-1.
      </a>
      <a
        class="category-link"
        href="/blog-category/category-2"
      >
        category-2.
      </a>
    </li>
    <li
      class="date"
    >
      2021/01/01
    </li>
  </ul>
</DocumentFragment>
`;
